tmt/frameworks/__init__.py

```diff
@@ -0,0 +1,119 @@
+from typing import TYPE_CHECKING, Callable, List, Type
+
+import tmt.log
+import tmt.plugins
+import tmt.result
+import tmt.utils
+
+if TYPE_CHECKING:
+    from tmt.base import Test
+    from tmt.steps.execute import ExecutePlugin
+    from tmt.steps.provision import Guest
+
+
+FrameworkClass = Type['ExecuteFramework']
+
+
+_FRAMEWORK_PLUGIN_REGISTRY: tmt.plugins.PluginRegistry[FrameworkClass] = \
+    tmt.plugins.PluginRegistry()
+
+
+def provides_framework(framework: str) -> Callable[[FrameworkClass], FrameworkClass]:
+    """
+    A decorator for registering export format.
```

Comment:
A copy paste typo? Should mention `registering execute framework` instead?

Reply:
Fixed in 76bc42ba.